File: src/main/java/com/example/testing_aop_proxy/entity/Statistic.java
package com.example.testing_aop_proxy.entity;

import lombok.Data;
import javax.persistence.*;

@Data
@Entity
public class Statistic {

    @Id
    @GeneratedValue(strategy = GenerationType.TABLE)
    private long id;

    @Column(nullable = false)
    private String className;

    @Column(nullable = false)
    private String method;

    @Column(nullable = false)
    private String requestId;

    private long duration;

    private long created = System.currentTimeMillis();
}

File: src/main/kotlin/com/example/testing_aop_proxy/entity/Statistic.kt
package com.example.testing_aop_proxy.entity

import javax.persistence.*

@Entity
data class Statistic(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        private val id: Long = 0,

        @Column(nullable = false)
        private val packageName: String,

        @Column(nullable = false)
        private val method: String,

        private val duration: Long = 0,
        private val created: Long = 0
)